index.js

```diff
@@ -60,6 +69,19 @@ module.exports = function (options, logger) {
       obj.headers = headers
     }
 
+    if (excludeHeaders && obj.headers) {
+      var headers = {}
+      Object.keys(obj.headers).forEach(function(name) {
+        headers[name] = obj.headers[name]
+      })
```

Comment:
We don't need to copy `obj.headers` both here and above in the `obscureHeaders` block if you are using both `obscureHeaders` and `excludeHeaders`. This can be merged to one operation.

Eg.
`if (obj.headers && (obscureHeaders || excludeHeaders)) {`

And then wrapping the for-loops with for `obscureHeaders and `excldueHeaders` in an if.